sentry-ruby/lib/sentry/transport/http_transport.rb

```diff
@@ -1,8 +1,13 @@
 require 'faraday'
+require 'zlib'
+require 'base64'
 
 module Sentry
   class HTTPTransport < Transport
-    CONTENT_TYPE = 'application/json'
+    GZIP_ENCODING = "gzip"
+    GZIP_THRESHOLD = 1024 * 30
```

Comment:
What is the "usual" size and throughput of Ruby events that you've seen so far? A threshold of somewhere around 32kB seems reasonable intuitively, I'd just love to confirm.

Reply:
In Rails apps, I think it's generally over 40kb because of the framework's stacktrace. But it could be quite small on non-Ruby apps, like 1x kb.